lib/Exception/ReceiptException.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+namespace Busuu\IosReceiptsApi\Exception;
+
+class ReceiptException extends \Exception {}
```

Comment:
no because InvalidArgumentException which extends LogicException and according to the documentation: 
> represents error in the program logic. This kind of exceptions should directly lead to a fix in your code